drracket/scribblings/tools/lang-tools.scrbl

```diff
@@ -74,98 +78,125 @@ The recognized token styles (specified implicitly via @method[color:text<%> star
           @item{@indexed-racket['parenthesis]}
           @item{@indexed-racket['error]}
           @item{@indexed-racket['other]}]
-These precise colors for these identifiers are controlled by the preferences dialog in DrRacket.
+
+The precise colors for these identifiers are controlled by the
+preferences dialog in DrRacket, and by other customization mechanisms
+in other tools.
+}
```

Comment:
The preferences are actually part of the framework/preferences library which has fairly minimal dependencies and, I believe, gives you back r/g/b values. Maybe it makes sense to elaborate a little bit here for other tool authors?

Reply:
The "The precise colors for these identifiers are controlled by the preferences dialog in Racket" text already existed. I just added ", and by other customization mechanisms in other tools."

For a tool like Emacs I think it makes sense to map tokens to Emacs named "faces" (e.g. "font-lock-string-face"); these faces in turn can be customized by the user or a theme chosen by the user. This way e.g. strings will look the same across various programming major modes.

So I haven't needed to dig into framework/preferences for this. I don't know enough to write the kind of explanation you want. Could you maybe add this yourself in a later commit once the rest of this looks OK and is merged?

Reply:
Ah, that makes sense. Sounds like the way it was is best, actually!

(I've read down beyond this but not quite finished reading the diff. Hopefully soon!)